### ISSUE TITLE
[Fixed in GH-6052] safari firefox: Missing Visual Focus on Registration Form

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4312]
![Screen Shot 2019-08-29 at 7 13 54 AM](https://user-images.githubusercontent.com/15113219/63949048-c51cd280-ca47-11e9-8d44-4009e8c443b3.png)

**Missing focus on:**

 - Select menu
 - Input field
 - Checkboxes
 - Buttons

**Comment by Zeyber** [comment]
Can see the focus throughout the form. Register button is difficult to see focus (raised an issue at #4991)

**Comment by KateChuen** [comment]
on Chrome ok.
These don't work: 
- the checkboxes on safari
- Register button on safari
- Terms & Conditions on firefox & safari
- I already have an account link on firefox & safari

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 1 02 35 PM](https://user-images.githubusercontent.com/15113219/73562899-0d428580-442a-11ea-97bc-4ab2dde3b5a7.png)
![Screen Shot 2020-01-31 at 1 02 44 PM](https://user-images.githubusercontent.com/15113219/73562909-14699380-442a-11ea-8c6c-ab1d53777654.png)
![Screen Shot 2020-01-31 at 1 02 49 PM](https://user-images.githubusercontent.com/15113219/73562915-192e4780-442a-11ea-9932-438e9a601d34.png)
![Screen Shot 2020-01-31 at 1 02 54 PM](https://user-images.githubusercontent.com/15113219/73562920-1d5a6500-442a-11ea-9ec8-0b399be0c1a8.png)

**Comment by KateChuen** [comment]
Tested on dev3
- Chrome
- Firefox
- Safari
